### PR TITLE
Json template

### PR DESCRIPTION
**Pull request by igor-ribeiiro**
Creating the code for what would be the JSON templates. Merging into a different branch other than master to not break existing code.

### CODE REVIEW

**Comment by igor-ribeiiro** [comment]
> Were you going to remove the WXP specific tests too (I mean as part of the branch)? They test shouldn't pass if they are still part of the test suite.

Yeah, just did, had forgotten previously

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
```
[the remark this answers is not in the document]
Comment (igor-ribeiiro) [reply manifest.json:1]: I don't think this `.json` is enough yet. But I wanted to have something there before we commit to creating a functional one
So this is more of a placeholder `.json` manifest for now

[the remark this answers is not in the document]
Comment (millerds) [reply manifest.json:1]: So are you planning on a further change to this repo to fill it out?  Is there a separate work item for that or is this part of the same work item (that won't be closed until it's done)?

File: package.json [hunk 2]
```diff
@@ -23,15 +23,13 @@
...
-    "test": "npm run test:unit && npm run test:e2e",
-    "test:e2e": "mocha -r ts-node/register test/end-to-end/*.ts",
-    "test:unit": "mocha -r ts-node/register test/unit/*.test.ts",
-    "validate": "office-addin-manifest validate manifest.xml",
+    "start": "office-addin-debugging start manifest.json",
+    "start:desktop": "office-addin-debugging start manifest.json desktop",
+    "start:desktop:outlook": "office-addin-debugging start manifest.outlook.json desktop --app outlook",
+    "start:web": "office-addin-debugging start manifest.json web",
+    "stop": "office-addin-debugging stop manifest.json",
+    "test": "",
```
Comment (millerds) [inline package.json:31]: If you look at other places we don't have tests (see scripts repo) then we use an "Echo" message indicating so.

**Review by millerds — CHANGES_REQUESTED** [review]
So you're not planning on making even a unit test for the template?

**Comment by igor-ribeiiro** [comment]
> So you're not planning on making even a unit test for the template?

What would the unit test be about? Those templates in general had unit and UI tests for WXP but no tests at all for Outlook.
So given this would be focused on outlook, I just removed the WXP tests

**Comment by igor-ribeiiro** [comment]
There is another work item for creating the valid json manifest:
"Create and test valid json manifest"
I like already pushing these changes as we won't depend on having the valid json in order to do the Yo Office work.

**Review by millerds — APPROVED** [review]

**Pull request merged**